schema.sql

```diff
@@ -7,4 +7,5 @@ CREATE TABLE animals (
     escape_attempts integer,
     neutered boolean,
     weight_kg decimal,
+    species varchar(255)
```

Comment:
- [ ] Great job so far. Adding a new column to the table, you are expected to use the SQL command for adding a new column to an existing table. You are expected to use the command below:
`ALTER TABLE animals ADD species varchar(255);`

Reply:
I used it, I happen to not have included it in the screenshot

Reply:
the screenshots indicate there is a species column.